packages/actions/src/uploadSplitDocumentation/index.ts

```diff
@@ -36,6 +58,9 @@ for await (const file of globber.globGenerator()) {
 
 try {
 	await Promise.all(promises);
+	if (failedUploads.length) {
+		setFailed(`Failed to upload ${failedUploads.length} files: ${failedUploads.join(', ')}`);
+	}
 } catch (error) {
 	console.log(error);
 }
```

Comment:
What about the `console.log(error);` here?

Reply:
I can do all the other places if you want (old docs upload and search indices for example), just felt like I was tackling one thing in particular here